server/lib/scanners/jellyfin/index.ts

```diff
@@ -210,14 +210,25 @@ class JellyfinScanner {
         return;
       }
 
-      if (metadata.ProviderIds.Tvdb) {
-        tvShow = await this.tmdb.getShowByTvdbId({
-          tvdbId: Number(metadata.ProviderIds.Tvdb),
-        });
-      } else if (metadata.ProviderIds.Tmdb) {
-        tvShow = await this.tmdb.getTvShow({
-          tvId: Number(metadata.ProviderIds.Tmdb),
-        });
+      if (metadata.ProviderIds.Tmdb) {
+        try {
+          tvShow = await this.tmdb.getTvShow({
+            tvId: Number(metadata.ProviderIds.Tmdb),
+          });
+        } catch {
+          this.log('No TMDb ID found for this title.', 'debug', {
+            jellyfinitem,
+          });
+        }
+      }
+      if (!tvShow && metadata.ProviderIds.Tvdb) {
+        try {
+          tvShow = await this.tmdb.getShowByTvdbId({
+            tvdbId: Number(metadata.ProviderIds.Tvdb),
+          });
+        } catch {
+          /* empty */
```

Comment:
Same here I guess 😅

Reply:
I added it [here](https://github.com/Fallenbagel/jellyseerr/pull/1138/commits/c164d97dde8b4b69891ea482e7b20cdcaaa3a553#diff-799b112b3397343d66cfa5140349c7551e63de0bebbf9bf628b7de831f916691R505-R507)

Reply:
Wait. It's not consistent. For TMDB it's logged only if there is an error while retrieving the tv show from jellyfin, while it's logged if there is a missing ID or and error for TVDB.
Do you think it makes sense to also log when there is no error but just a missing ID or not?

Reply:
I'd say we should log both? Not sure how you guys handle logging though